conf.d/fish_ai.fish

```diff
@@ -8,24 +8,54 @@ set -g install_dir (test -z "$XDG_DATA_HOME"; and echo "$HOME/.local/share/fish-
 set -g config_path (test -z "$XDG_CONFIG_HOME"; and echo "$HOME/.config/fish-ai.ini"; or echo "$XDG_CONFIG_HOME/fish-ai.ini")
 
 ##
-## This section contains the keybindings for fish-ai. If you want to change the
-## default keybindings, use the environment variables:
+## This section contains the keybindings for fish-ai.
+##  keymap_1 - codify or explain, defaults to Ctrl + P
+##  keymap_2 - autocomplete or fix, defaults to Ctrl + Space
 ##
-##   - FISH_AI_KEYMAP_1 (defaults to Ctrl + P)
-##   - FISH_AI_KEYMAP_2 (defaults to Ctrl + Space)
+## If you want to change the default keybindings, add this to your `fish-ai.ini`:
+##  ```
+##  [fish-ai]
+##  keymap_1 = <your keybinding>
+##  keymap_2 = <your keybinding>
+##  ```
+## or use the environment variables:
+##
+##   - FISH_AI_KEYMAP_1
+##   - FISH_AI_KEYMAP_2
 ##
 ## These should be set to the key binding escape sequence for a keyboard shortcut
 ## you want to use + any flags. You can get the key binding escape sequence using
 ## the command `fish_key_reader`.
 ##
-if test -n "$FISH_AI_KEYMAP_1"
-    set -g keymap_1 $FISH_AI_KEYMAP_1
-else
+## Note: shell restart is required for new keybindings to take effect.
+##
+function _fish_ai_get_conf_value -a key
```

Comment:
There is code for reading from the configuration file in the Python module. For example, `"$install_dir/bin/lookup_setting" "keymap_1"`.

The string would then have to be unescaped, e.g. using [`string unescape`](https://fishshell.com/docs/current/cmds/string-unescape.html).